### PR TITLE
VULCAN-91 Rewording

### PR DESCRIPTION
**Pull request by nhpatt**
Ping :) @ngaskill

### CODE REVIEW

File: README.markdown [hunk 1]
```diff
@@ -1,50 +1,48 @@
-# Vulcan, may your APIs live long and prosper
+# Vulcan Architect, may your APIs live long and prosper
```
Comment (JorgeFerrer) [inline README.markdown]: Tenemos que quitar la frase "may your APIs live long and prosper"

File: README.markdown [hunk 2]
```diff
@@ -1,50 +1,48 @@
...
 
-## The Vulcan project
-
-The Vulcan project provides a set of Guidelines and Software to build APIs and consumers designed to evolve.
+# [Vulcan Architect](https://github.com/liferay/com-liferay-vulcan)
 
-### [Vulcan Guidelines](https://vulcan.wedeploy.io/guidelines/)
+Vulcan Architect is part of the Vulcan project. Here is a short description of what the project is about:
```
Comment (JorgeFerrer) [inline README.markdown]: Esto ha quedado un poco raro, cuando sugerí añadir la descripción del proyecto me refería al proyecto vulcan. Propongo cambiarlo por:

Vulcan Architect is part of the Vulcan project, which aims to promote the creation of APIs designed to evolve over time. The project also provides a set of guidelines for API providers and consumers that can be implemented in any technology and Vulcan Consumer, a library to facilitate the creation of consumers of any hypermedia API.

File: README.markdown [hunk 3]
```diff
@@ -1,50 +1,48 @@
...
+## How will it help me?
 
-With Vulcan Architect you can create APIs that follow all the principles of Vulcan Guidelines without much effort.
+Architect provides JAX-RS writers for the most important Hypermedia formats, such as HAL or JSON-LD (with more coming). It also provides an easy way of representing your resources, in a really generic way, so every representation can understand it, but following common Hypermedia patterns, such as the Representor.
 
-### Vulcan Consumers
+Also provides a really simple way of creating the different endpoints for your resources, which has many similarities with the JAX-RS approach. So migrating your API from a REST JAX-RS implementation to Architect will be easy as pie.
 
-A client-side library to facilitate developing consumers that consume Vulcan REST APIs (or any Hypermedia API, really). It also has some smart capabilities such as automatic creation of a local graph to facilitate building offline support.
+## What if I just want to try these "Vulcan APIs"?
```
Comment (JorgeFerrer) [inline README.markdown]: Cambiar pregunta a algo como "Can I try a sample Vulcan API online?"

Comment (ahdezma) [reply README.markdown]: 👍

File: README.markdown (hunk 2 above)
Comment (ahdezma) [reply README.markdown]: 👍

**Comment by ahdezma** [comment]
ci:retest

**Pull request merged**